contracts/profit/src/state/buy_back.rs

```diff
@@ -140,7 +141,11 @@ impl SwapTask for BuyBack {
 }
 
 impl ConfigManagement for StateLocalOut<BuyBack, ForwardToDexEntry, ForwardToDexEntryContinue> {
-    fn try_update_config(self, _: CadenceHours) -> ContractResult<Self> {
+    fn try_update_config(
+        self,
+        _: Timestamp,
+        _: CadenceHours,
+    ) -> ContractResult<StateMachineResponse<Self>> {
         Err(ContractError::UnsupportedOperation(String::from(
             "Configuration changes are not allowed during ICA opening process.",
         )))
```

Comment:
why don't we provide such an implementation at the trait level? While overwriting an implementation is not acceptable, providing a default rejecting logic saves code duplication.

Reply:
I admit that Ctrl+C/V is not a good approach, but this was somewhat of a dirty refactor to get the structure look better.

Side question: Why is not acceptable to override a default method?
Side note: If that's the case, then I would suggest having an `*Ext` trait which has blanket implementations for all of our traits that currently do have default implementations which shouldn't be changed.